README.md

```diff
@@ -1,8 +1,55 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Welcome to Mutual-Aid-Project! This site is for organizing mutual aid efforts within your community. 
 
 ## Setup
 
-Environmental variables:  
+Before starting, please reach out to Megan at info@townhallproject.com for environmental variables. 
+Once you receive this information please create an .env file at the root level (same level as .gitignore) 
+and copy and paste information into that file. 
+
+#### The general development workflow is as follows:
+- Fork the Mutual-Aid-project admin repo
```

Comment:
Fork the Mutual-Aid-project admin repo -> Fork the Mutual-Aid-Networks repo

Reply:
updated